src/app/components/MediaPlayer/AudioLoader/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { node } from 'prop-types';
+import styled from '@emotion/styled';
+import { keyframes } from '@emotion/react';
+
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+const Overlay = styled.div`
+  .no-js & {
+    display: none;
+  }
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+`;
+
+const spin = keyframes`
+0% {
+  transform: rotate(0deg);
+}
+100% {
+  transform: rotate(360deg);
+}
+`;
+
+const Spinner = styled.div`
+  width: 5rem;
+  height: 5rem;
+  border-radius: 50%;
+  border: 0.3rem solid rgba(0, 0, 0, 0.2);
+  border-left-color: rgba(0, 0, 0, 0.8);
+  animation: ${spin} 1.1s infinite linear;
```

Comment:
There's an example of a reduced motion media-query we can use for that requirement of the AAC https://github.com/bbc/elections-web-components-us2020/blob/main/src/components/banner/live-pulse.jsx#L27

Reply:
I think we also need to consider this https://paper.dropbox.com/doc/Animation-Decorative--BIclH_cASo5r3CQ2RKkPnvaTAg-n3x1bxFBYh0RQx73W9OX3

Reply:
Cool - didn't know that reduced-motion media query was a thing